src/inputs/OptionSelect.tsx

```diff
@@ -35,14 +36,26 @@ class OptionSelect extends Component<IOptionSelectProps> {
     return getOptions(this.fieldConfig, this.injected);
   }
 
+  private get showSearch (): boolean {
+    // 8 is the most number of options you can show with no scroll
+    const SHOW_SEARCH_IF_OVER = 8;
```

Comment:
Do we normally define constants like these in functions? I thought we put them at the top of the file generally, even if they are only used once in the component